intermediate_source/char_rnn_classification_tutorial.py

```diff
@@ -3,6 +3,7 @@
 NLP From Scratch: Classifying Names with a Character-Level RNN
 **************************************************************
 **Author**: `Sean Robertson <https://github.com/spro>`_
+**Updated**: `Matthew Schultz <https://github.com/mgs28>`_ 
```

Comment:
we typically don't add Update. Please remove this

Reply:
done!

Reply:
@svekars  - anything else I can do to make this better? thanks!

Reply:
@svekars - hello, are there other items I should address here? I appreciate your help with this!